chain/network/src/config.rs

```diff
@@ -386,6 +389,15 @@ impl NetworkConfig {
                 self.peer_recent_time_window, UPDATE_INTERVAL_LAST_TIME_RECEIVED_MESSAGE
             );
         }
+
+        if !(self.max_send_peers <= PEERS_RESPONSE_MAX_PEERS) {
```

Comment:
nit: can just use > ?

Reply:
Leaving this for consistency with the existing checks in this function:

https://github.com/near/nearcore/blob/bfe384697b0482e0a77085fc74c053e61fd87ffa/chain/network/src/config.rs#L363